bitbots_body_behavior/bitbots_body_behavior/decisions/reached_goal.py

```diff
@@ -8,16 +8,16 @@
     AbstractDecisionElement
 
 
-class ReachedMovebaseGoalPosition(AbstractDecisionElement):
+class ReachedGoalPosition(AbstractDecisionElement):
```

Comment:
Maybe call it something like ReachedPathPlanningGoalPosition. Just Goal might be confused with the Goal Object in the soccer context.